Observer/Config/CronFrequency.php

```diff
@@ -35,7 +35,10 @@ class CronFrequency
     protected $cronFrequency = [
         'catalog_sync_frequency' => [
             'config_path' => '',
-            'job_code' => 'yotpo_cron_core_products_sync,yotpo_cron_core_category_sync,yotpo_cron_core_collections_products_sync'
+            'job_code' =>
```

Comment:
Is breaking the string to multiple lines here allowed?

Reply:
Sure, it is fine to break them into new lines. That newline character won't have any presence in that data.